eng/cibuild_bootstrapped_msbuild.ps1

```diff
@@ -123,6 +124,9 @@ try {
   # Ensure that debug bits fail fast, rather than hanging waiting for a debugger attach.
   $env:MSBUILDDONOTLAUNCHDEBUGGER="true"
 
+  # Opt into performance logging. https://github.com/dotnet/msbuild/issues/5900
+  $env:DOTNET_PERFLOG_DIR=$PerfLogDir
```

Comment:
Setting this at this layer means it applies only to the bootstrapped build. That's mostly fine, but it might be nice to have the perf data for the Windows/Release build too, which doesn't run the bootstrap. Worth changing? To argue with myself for a second, maybe not because it'd always be collecting the data from the LKG MSBuild installed on the build agent which is maybe not interesting.

Reply:
(I happened to check back in on this right as this was posted)

I considered that but ultimately came to the same conclusion. I can envision a future where we capture both and run a tool to diff the two so we can have our own mini-RPS, but I'll save that for when we decide to prioritize something like that.